__tests__/index.test.js

```diff
@@ -0,0 +1,33 @@
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const request = require('supertest')
+const app = require("../app")
+const data = require('../db/data/test-data/index')
+
+beforeEach(() => {
+   return seed(data) 
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe("/api/topics", () => {
+    test("responds with all topics", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+            expect(response.body.topics.length).toBe(3);
+            response.body.topics.forEach((topic) => {
+              expect(typeof topic.description).toBe('string');
+              expect(typeof topic.slug).toBe('string');
+            })
+        })
+    })
+    test("responds with all topics", () => {
+        return request(app)
+        .get("/api/topicsss")
+        .expect(404)
+    })
```

Comment:
I like the idea behind this test but it isn't needed as it will auto pass using Express default behaviour. However, you could keep it if you want it to also send an error message, like `endpoint not found` You don't have to do this though! If you do decide you are happy with the default behaviour, please do delete this test 😄